src/System.Reflection.Context/src/System.Reflection.Context.csproj

```diff
@@ -5,7 +5,7 @@
     <OutputType>Library</OutputType>
     <ProjectGuid>{404DB891-B5AF-41E6-B89D-29E3F4573C4F}</ProjectGuid>
     <!-- only supported by desktop CLR -->
-    <GeneratePlatformNotSupportedAssembly>true</GeneratePlatformNotSupportedAssembly>
+    <GeneratePlatformNotSupportedAssemblyMessage>Customized reflection contexts are only supported on full .NET framework.</GeneratePlatformNotSupportedAssemblyMessage>
```

Comment:
Nit: sometimes you've capitalized Framework, sometimes not.  Do we have crisp guidance on when we say "full .NET Framework", ".NET Framework", "desktop", etc.?